internal/rbd/rbd_util.go

```diff
@@ -229,6 +230,16 @@ func createImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 		return fmt.Errorf("failed to create rbd image: %w", err)
 	}
 
+	if pOpts.ThickProvision {
+		err = pOpts.allocate(ctx)
```

Comment:
Any rough figures on how much allocation time different size rbd image can take?

Reply:
Not much. I expect this to be the same performance as the `rbd create --thick-provision` option. It will be slow, as writing data to the image needs to be done. @gobindadas might be able to share some statistics from that.

Reply:
one last thing we need to check is that, when the user request for pvc with a greater size say 100 GB. as this is going to take time, meanwhile what happens if the user deletes the pvc. will that leave any stale PV or rbd image?